crates/astria-composer/src/executor/tests.rs

```diff
@@ -112,15 +115,37 @@ async fn setup() -> (MockServer, MockGuard, Config, NamedTempFile) {
         grpc_addr: "127.0.0.1:0".parse().unwrap(),
         fee_asset: "nria".parse().unwrap(),
     };
-    (server, startup_guard, cfg, keyfile)
+    (server, cfg, keyfile)
+}
+
+/// Assert that given error is of correct type and contains the expected chain IDs.
+#[track_caller]
+fn assert_chain_id_err(
+    err: &EnsureChainIdError,
+    configured_expected: &str,
+    configured_actual: &tendermint::chain::Id,
+) {
+    match err {
+        EnsureChainIdError::WrongChainId {
+            expected,
+            actual,
+        } => {
+            assert_eq!(*expected, configured_expected);
+            assert_eq!(*actual, *configured_actual);
+        }
+        other @ EnsureChainIdError::GetChainId(_) => {
+            panic!("expected `EnsureChainIdError::WrongChainId`, but got '{other}'")
```

Comment:
Last change before merge:
```suggestion
            panic!("expected `EnsureChainIdError::WrongChainId`, but got '{other:?}'")
```
^ I'd have done it myself but don't have push rights for your fork.